app/components/edit/tab_form/next_button_component.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
 
-module Elements
+module Edit
```

Comment:
The tab form got moved to `edit/` instead of `elements/` because it is tied to the work / collection forms, i.e., it isn't really generic.